File: pert4/tugas.cpp
#include <iostream>

using namespace std;

void tambah(int angka1, int angka2){
    cout << "TAMBAH : " << angka1 + angka2 <<endl;
}
void kurang(int angka1, int angka2){
    cout << "KURANG" << angka1 - angka2 <<endl;
}

void kali(int angka1, int angka2){
    cout << "KALI" << angka1 * angka2 <<endl;
}

void bagi(int angka1, int angka2){
    cout << "BAGI" << angka1 / angka2 <<endl;
}

int main(){
    int angka1, angka2;
    char pilih;
    do{
        cout << "MENU" << endl;
        cout << "1. Tambah" << endl;
        cout << "2. Kurang" << endl;
        cout << "3. Kali" << endl;
        cout << "4. Bagi" << endl;
        cout << "Pilihan : ";
        cin >> pilih;

        switch (pilih){
            case '1':
                cout << "Masukkan Angka 1 : ";
                cin >> angka1;
                cout << "Masukkan Angka 2 : ";
                cin >> angka2;
                tambah(angka1, angka2);
                break;
            case '2':
                cout << "Masukkan Angka 1 : ";
                cin >> angka1;
                cout << "Masukkan Angka 2 : ";
                cin >> angka2;
                kurang(angka1, angka2);
                break;
            case '3':
                cout << "Masukkan Angka 1 : ";
                cin >> angka1;
                cout << "Masukkan Angka 2 : ";
                cin >> angka2;
                kali(angka1, angka2);
                break;
            case '4':
                cout << "Masukkan Angka 1 : ";
                cin >> angka1;
                cout << "Masukkan Angka 2 : ";
                cin >> angka2;
                bagi(angka1, angka2);
                break;
            case '5':
                cout << "EXIT" << endl;
                break;
            default:
                cout << "Pilihan Salah!" << endl;
        }
    }while(pilih != '5');
    return 0;
}

File: pert4/tugas.py
def get_int(prompt):
    while True:
        try:
            return int(input(prompt))
        except ValueError:
            print("Masukkan Angka")

def tambah(a,b):
    print("tambah:",a + b)

def kurang(a,b):
    print("kurang:",a - b)

def kali(a,b):
    print("kali:",a * b)

def bagi(a,b):
    print("bagi:",a / b)


def main():
    while True:
        print("MENU")
        print("1. Tambah")
        print("2. Kurang")
        print("3. Kali")
        print("4. Bagi")
        print("5. Exit")

        pilih = input("Pilihan : ")

        if pilih == '5':
            print ("Exit")
            break
        if pilih in ('1','2','3','4'):
            a = get_int("Masukkan Angka 1 : ")
            b = get_int("Masukkan Angka 2 : ")

            if pilih == '1':
                tambah(a,b)
            elif pilih == '2':
                kurang(a,b)
            elif pilih == '3':
                kali(a,b)
            elif pilih == '4':
                bagi(a,b)
            else:
                print("Pilihan Salah")

if __name__ == "__main__":
     main()
            